BraveWallet/WalletHostingViewController.swift

```diff
@@ -15,6 +15,8 @@ public protocol BraveWalletDelegate: AnyObject {
   ///
   /// This will be called after the wallet UI is dismissed
   func openWalletURL(_ url: URL)
+  /// Will dismiss wallet panel and open full screen wallet or full screen wallet settings
+  func onFullScreenWallet(_ viewController: UIViewController)
```

Comment:
Naming needs work, you're saying to full screen wallet but you pass an arbitrary `UIViewController`. Overall this is confusing.

Reply:
Also please avoid prefixing methods with "on"